public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap

```diff
@@ -149,15 +149,17 @@ exports[`Dashboard renders in flyout 1`] = `
               <div
                 className="euiFlexItem"
               >
-                <EuiTitle
-                  size="l"
+                <EuiText
+                  size="s"
                 >
-                  <h3
-                    className="euiTitle euiTitle--large"
+                  <div
+                    className="euiText euiText--small"
```

Comment:
you can just drop these classes and the div - they aren't necessary

Reply:
those are automatically updated snapshot files. If we remove it, test will fail, CI will fail
![image](https://github.com/user-attachments/assets/8382e773-f73b-4503-97f2-dbab2a326f2c)

Reply:
Ah, missed that. Looks like ContentPanel is doing something weird, but don't have to go deeper to fix